src/Lynx/Form.purs

```diff
@@ -30,17 +30,19 @@ import Test.QuickCheck (class Arbitrary)
 import Test.QuickCheck.Arbitrary (genericArbitrary)
 import Type.Row (type (+))
 
-type LayoutRows c r =
-  ( name :: String
-  , contents :: NonEmpty Array c
-  | r
-  )
 
-type Page f = Record (LayoutRows (Tab f) ())
+type Page f = { name :: String
+              , tabs :: NonEmpty Array (Tab f)
```

Comment:
We should recognize that changing any of these fields is a breaking change.

Since we rely on the instances for decoding/encoding that depend on the labels of a record and because we store `Page _`s directly as those JSON blobs in OMS, this means that we'll not be able to parse older versions of `Page _` going forward. E.g. all of the data in all current deployments of OMS will be no longer be parseable.

Nothing is live, so we can probably just break stuff. But, we might want to think about no longer storing data as JSON blobs in OMS sooner rather than later. Either that or thinking about migrating through different versions.

Reply:
I agree that we can probably just blow away what we have in OMS right now, but think we need a longer-term migration strategy as well.